src/components/HomePage.js

```diff
@@ -120,6 +120,15 @@ var unlinkable_clients = [
         maturity: "Alpha",
         comments: "Access any room anywhere in Matrix via good old IRC!",
     },
+    {
+        name: "uMatriks",
+        logo: "img/uMatriks-256px.png",
+        author: "LarreaMikel",
+        homepage: "https://github.com/LarreaMikel/uMatriks",
+        maturity: "Alpha",
+        room_instructions(userId)  { return <span>Click "Join room"</span> },
```

Comment:
the fn signature should be `room_instructions(alias)` but what I meant is that the instruction seems really useless, doesn't even mention the alias to join, look at the instruction for PTO for instance `room_instructions(alias)  { return <span>Type <code>/join <b>{ alias }</b></code></span> },`